preload/link-header-preload-non-html.html

```diff
@@ -2,11 +2,13 @@
 <meta charset=utf-8>
 <title>Makes sure that Link headers preload resources in non-HTML documents</title>
 <meta name="timeout" content="long">
+<meta name="pac" content="/proxy.pac">
```

Comment:
I think there should at least be an example test in `infrastructure` to verify that the pac support works. The manifest changes would probably also benefit from some pytest tests in `tools/manifest/tests`.